src/daemon/bitcoind/poller.rs

```diff
@@ -611,6 +611,9 @@ fn maybe_cpfp_txs(
         .collect();
 
     // TODO: std transaction max size check and split
+    // FIXME: to_cpfp might include some transactions that aren't confirmed, but
+    // have already been CPFPed. If that's the case, cpfp_package will fail and
+    // won't CPFP any transaction.
```

Comment:
This is in essence the same FIXME as the above:
>   // FIXME: it's a shame to have to get the derivation paths from bitcoind, but we need to
>   // for the Spend CPFP output derivation index. We have all the info and should be able to
>  // make it work without this hack.

Reply:
But yea it didn't mention RBF (because i wanted to have #333 merged before but hey)

Reply:
> This is in essence the same FIXME as the above:

No? My FIXME says that if we include in to_cpfp a transaction whose cpfp output was already spent, we fail to cpfp everything. Your comment isn't saying that?

Reply:
Your FIXME is fixed by mine is what i was saying, but nevermind it's really not important :)